workshops/forms.py

```diff
@@ -146,49 +145,32 @@ def __init__(self, *args, **kwargs):
         self.helper.form_method = 'get'
         self.helper.layout = Layout(
             Div(
-                Div(
-                    'latitude',
-                    'longitude',
-                    css_class='panel-body'
-                ),
-                css_class='panel panel-default ',
-            ),
-            HTML('<p>OR</p>'),
-            Div(
-                Div(
-                    'airport',
-                    css_class='panel-body'
-                ),
-                css_class='panel panel-default ',
-            ),
-            HTML('<p>OR</p>'),
-            Div(
-                Div(
-                    'country',
-                    css_class='panel-body'
-                ),
+                Div(HTML('Location close to'), css_class='panel-heading'),
+                Div('airport', css_class='panel-body'),
+                Div(HTML('<b>OR</b>'), css_class='panel-footer'),
+                Div('country', css_class='panel-body'),
                 css_class='panel panel-default ',
             ),
+            'instructor_badges',
+            'was_helper',
+            'was_organizer',
+            'is_in_progress_trainee',
             'gender',
             'lessons',
-            'instructor_badges',
             FormActions(
                 Submit('submit', 'Submit'),
             ),
         )
 
     def clean(self):
-        cleaned_data = super(InstructorsForm, self).clean()
+        cleaned_data = super().clean()
         airport = cleaned_data.get('airport')
-        lat = cleaned_data.get('latitude')
-        long = cleaned_data.get('longitude')
         country = cleaned_data.get('country')
 
-        sum = bool(airport) + bool(lat and long) + bool(country)
-        # user can specify only one: either airport, or lat&long, or country
-        if sum != 1:
-            raise forms.ValidationError('Must specify an airport, or latitude'
-                                        ' and longitude, or a country.')
+        # user can specify only one: either airport or country
+        if not (bool(airport) ^ bool(country)):
+            raise forms.ValidationError('Must specify an airport OR a '
+                                        'country.')
```

Comment:
User can specify only one *or none*, i.e. searching for all SWC instructors is a valid query.